Problem: android integration test fails

e.g. https://github.com/crypto-com/defi-wallet-core-rs/runs/8195610535?check_suite_focus=true
```
Loading local repository...                                                     
697
[                                       ] 3% Loading local repository...        
698
[                                       ] 3% Fetch remote repository...         
699
[                                       ] 3% Fetch remote repository...         
700
Warning: File /Users/runner/.android/repositories.cfg could not be loaded.
701
                                                                                
702
[                                       ] 3% Fetch remote repository...         
703
[=                                      ] 4% Fetch remote repository...         
704
[=                                      ] 4% Fetch remote repository...         
705
[=                                      ] 4% Fetch remote repository...         
706
[=                                      ] 5% Fetch remote repository...         
707
[=                                      ] 5% Fetch remote repository...         
708
[=                                      ] 5% Fetch remote repository...         
709
[=                                      ] 5% Fetch remote repository...         
710
[=                                      ] 5% Fetch remote repository...         
711
[=                                      ] 5% Fetch remote repository...         
712
[=                                      ] 5% Fetch remote repository...         
713
[=                                      ] 5% Fetch remote repository...         
714
[=                                      ] 5% Fetch remote repository...         
715
[==                                     ] 5% Fetch remote repository...         
716
[==                                     ] 5% Fetch remote repository...         
717
[==                                     ] 5% Fetch remote repository...         
718
[==                                     ] 5% Fetch remote repository...         
719
[==                                     ] 5% Fetch remote repository...         
720
[==                                     ] 5% Fetch remote repository...         
721
[==                                     ] 5% Fetch remote repository...         
722
[==                                     ] 5% Fetch remote repository...         
723
[==                                     ] 5% Fetch remote repository...         
724
[==                                     ] 6% Fetch remote repository...         
725
[==                                     ] 6% Fetch remote repository...         
726
[==                                     ] 6% Fetch remote repository...         
727
[==                                     ] 6% Fetch remote repository...         
728
[==                                     ] 6% Fetch remote repository...         
729
[==                                     ] 6% Fetch remote repository...         
730
[==                                     ] 6% Fetch remote repository...         
731
Warning: Failed to find package system-images;android-;google_apis;x86_64
732
[==                                     ] 6% Fetch remote repository...         
733
[==                                     ] 6% Fetch remote repository...         
734
[==                                     ] 6% Fetch remote repository...         
735
[==                                     ] 6% Computing updates...               
736
[===                                    ] 8% Computing updates...               
737
[===                                    ] 8% Computing updates...               
738
[===                                    ] 10% Computing updates...              
739
                                                                                
740
[===                                    ] 10% Computing updates...              
741
Error: Process completed with exit code 1.
```

--- a/.github/workflows/android-build.yml
+++ b/.github/workflows/android-build.yml
@@ -21,10 +21,8 @@ jobs:
       - uses: actions/checkout@v2
       - run: |
           export JAVA_HOME=$JAVA_HOME_8_X64
-          export ANDROID_VERSION=`$ANDROID_SDK_ROOT/tools/bin/sdkmanager --list 2> /dev/null | tail -n 5 | grep platform-tools | awk '{print $3}' | awk -F. '{print $1}'`
-          echo $ANDROID_VERSION
-          echo y | $ANDROID_SDK_ROOT/tools/bin/sdkmanager "system-images;android-$ANDROID_VERSION;google_apis;x86_64"
-          echo no | $ANDROID_SDK_ROOT/tools/bin/avdmanager create avd -n testavd -c 2048M -k "system-images;android-$ANDROID_VERSION;google_apis;x86_64"
+          echo y | $ANDROID_SDK_ROOT/tools/bin/sdkmanager "system-images;android-31;google_apis;x86_64"
+          echo no | $ANDROID_SDK_ROOT/tools/bin/avdmanager create avd -n testavd -c 2048M -k "system-images;android-31;google_apis;x86_64"
           $ANDROID_SDK_ROOT/tools/emulator @testavd -partition-size 2048 -wipe-data &
       - name: NDK cache
         uses: actions/cache@v2

